packages/web-app/src/lib/models/placeDetails.ts

```diff
@@ -140,38 +96,36 @@ const buildPublics = (publics: Publics): PlaceDetailsInfo[] => {
     [WelcomedPublics.PREFERENTIAL]: PlaceDetailsInfoType.WELCOME_UNCONDITIONAL_CUSTOM,
     [WelcomedPublics.EXCLUSIVE]: PlaceDetailsInfoType.WELCOME_EXCLUSIVE
   };
-
-  const description =
-    publics.accueil === WelcomedPublics.UNCONDITIONAL
-      ? []
-      : [
-          ...buildSpecificPublics(publics.gender, GENDER_DEFAULT_VALUES, PUBLICS_LABELS.gender),
-          ...buildPublicsAge(publics.age.min, publics.age.max),
-          ...buildSpecificPublics(
-            publics.administrative,
-            ADMINISTRATIVE_DEFAULT_VALUES,
-            PUBLICS_LABELS.administrative
-          ),
-          ...buildSpecificPublics(
-            publics.familialle,
-            FAMILY_DEFAULT_VALUES,
-            PUBLICS_LABELS.familialle
-          ),
-          ...buildSpecificPublics(publics.other, OTHER_DEFAULT_VALUES, PUBLICS_LABELS.other)
-        ];
-
-  return [
-    { type: WELCOME_TO_TITLE_MAPPING[publics.accueil], tags: [], description },
-    ...(publics.description
-      ? [
-          {
-            type: PlaceDetailsInfoType.PUBLICS_MORE_INFO,
-            tags: [],
-            description: [{ key: publics.description }]
-          }
-        ]
-      : [])
+  const currentLang = getCurrentLangInStorage();
+
+  const description = translatePublics(
+    i18nInstance,
+    currentLang as unknown as SupportedLanguagesCode,
+    publics,
+    true,
+    false
+  );
+
+  const publicsInfo: PlaceDetailsInfo[] = [
+    {
+      type: WELCOME_TO_TITLE_MAPPING[publics.accueil],
+      tags: [],
+      description: [],
+      translatedText: description,
+      needTranslation: false
+    }
   ];
+
+  if (publics?.description) {
+    // eslint-disable-next-line fp/no-mutating-methods
+    publicsInfo.push({
+      type: PlaceDetailsInfoType.PUBLICS_MORE_INFO,
+      tags: [],
+      description: [{ key: publics?.description }],
+      needTranslation: true
+    });
+  }
+  return publicsInfo;
```

Comment:
Je veux bien qu'on garde un truc de programmation fonctionnelle un peu débile pour le moment stp :)
```suggestion
  const currentLang = getCurrentLangInStorage();

  const description = translatePublics(
    i18nInstance,
    currentLang as unknown as SupportedLanguagesCode,
    publics,
    true,
    false
  );

  return [
    { 
      type: WELCOME_TO_TITLE_MAPPING[publics.accueil],
      tags: [], 
      description: [],
      translatedText: description,
      needTranslation: true
     },
    ...(publics.description
      ? [
          {
            type: PlaceDetailsInfoType.PUBLICS_MORE_INFO,
            tags: [],
            description: [{ key: publics.description }],
            needTranslation: true
          }
        ]
      : [])
```